src/Symfony/Bundle/FrameworkBundle/Resources/config/annotations.php

```diff
@@ -22,7 +22,7 @@
 return static function (ContainerConfigurator $container) {
     $container->services()
         ->set('annotations.reader', AnnotationReader::class)
-            ->call('addGlobalIgnoredName', ['required']) // @deprecated since Symfony 6.3
+            ->call('addGlobalIgnoredName', ['required'])
```

Comment:
keeping this as is so that ppl can still use the annotation without being bothered